examples/step-75/step-75.cc

```diff
@@ -669,8 +669,7 @@ namespace Step75
           dof_handler.get_triangulation());
     else
       coarse_grid_triangulations.emplace_back(
-        const_cast<Triangulation<dim> *>(&(dof_handler.get_triangulation())),
-        [](auto &) {});
+        &(dof_handler.get_triangulation()), [](auto) {});
```

Comment:
We could switch to `auto *` if this is deemed to be more elegant. @marcfehling @peterrum

Reply:
Let's do it consistently: https://github.com/dealii/dealii/blob/e6779913e7f770652da442b3c2460dec3e864029/include/deal.II/multigrid/mg_transfer_global_coarsening.templates.h#L2377-L2378

Reply:
Fixed.